harness/tests/fixtures/api_responses.py

```diff
@@ -50,6 +50,12 @@ def sample_get_experiment_trials() -> bindings.v1GetExperimentTrialsResponse:
         return resp
 
 
+def sample_get_experiment_checkpoints() -> bindings.v1GetExperimentCheckpointsResponse:
+    with open(FIXTURES_DIR / "experiment_checkpoints.json") as f:
```

Comment:
Could you walk me through the justification to make this `experiment_checkpoints.json` over `checkpoints.json`? Could the file represent any paginated collection of checkpoints?

Reply:
yeah good point, updated